Makefile

```diff
@@ -537,4 +556,26 @@ contracts/test/prover/proofs/%.json: $(arbitrator_cases)/%.wasm $(prover_bin)
 
 always:              # use this to force other rules to always build
 .DELETE_ON_ERROR:    # causes a failure to delete its target
-.PHONY: push all build build-node-deps test-go-deps build-prover-header build-prover-lib build-prover-bin build-jit build-replay-env build-solidity build-wasm-libs contracts format fmt lint stylus-benchmarks test-go test-gen-proofs push clean docker
+.PHONY: all
+.PHONY: build
```

Comment:
One thing I usually do to improve the phony declaration is putting it alongside the target. For instance:
```
.PHONY: build
build:
	...
```

Reply:
Thanks for the idea. Done.